src/components/AddProducts/AddProduct.tsx

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+import { TextField, FormLabel, Typography, Select, Grid, Input, Box, Paper, Button } from '@material-ui/core';
+import { useStyles } from './AddProduct.styles';
+import CloudUploadIcon from '@material-ui/icons/CloudUpload';
+import useTexts from '../../hooks/useTexts';
+
+const AddProduct: React.FC = () => {
+  const classes = useStyles();
+  const texts = useTexts();
+  const [categoryName, setCategoryName] = React.useState<string[]>([]);
+
+  const handleChange = (event: React.ChangeEvent<{ value: unknown }>) => {
+    setCategoryName(event.target.value as string[]);
+  };
+  return (
+    <div className={classes.root}>
+      <Paper className={classes.paper}>
+        <FormLabel component="legend">
+          <Typography variant="h4" gutterBottom>
+            {texts.addProductFormTitle}
+          </Typography>
+        </FormLabel>
+        <div>
+          <Grid container spacing={3}>
+            <Grid item xs={3}>
+              <label>{texts.addProductProductLabel}</label>
+            </Grid>
+            <Grid item xs={6}>
+              <TextField
```

Comment:
https://material-ui.com/components/text-fields/#textfield -> the normal variant of TextField is no longer in the material guidelines, we should add variant="outlined" to all textfields